packages/gradle-plugin/build.gradle.kts

```diff
@@ -106,8 +115,10 @@ tasks.create("pluginVersion") {
             // Generated file. Do not edit!
             package io.realm.kotlin.gradle
             internal const val PLUGIN_VERSION = "${project.version}"
+            internal const val CORE_VERSION = "${coreVersion}"
```

Comment:
It is not tricky ... on the client side. But it will probably clutter the remote view. I guess we should rather just strive to use tagged core-releases 😉 For discussion see https://mongodb.slack.com/archives/C04M17MCY0H/p1693920525143069